### ISSUE TITLE
Is it possible to please remove the need for, or add the option to disable "Console.Readline();" when patching to enable easy batch processing of patches?

### ISSUE THREAD

**Issue by PerryTute** [AntonioDePau/KHPCPatchManager#32]
It's very frustrating and I'm unfortunately not well versed enough in C# to figure out how to get this damn requirement out of the way!

**Comment by AntonioDePau** [comment]
Hey, why not apply multiple patches at once?

**Comment by PerryTute** [comment]
> Hey, why not apply multiple patches at once?

Sorry about my initial post, had a case of 3am brain going on, what I meant to say is applying multiple patches at once is a bit tedious as at the end of each patching process the "enter" key is require to begin the processing of the next patch process, so for example:

`patch.exe khpcpatchmanager kh1_fifth.kh1pcpatch
"ENTER"
patch.exe khpcpatchmanager kh1_first.kh1pcpatch
"ENTER"
patch.exe khpcpatchmanager kh1_fourth.kh1pcpatch
"ENTER"
patch.exe khpcpatchmanager kh1_second.kh1pcpatch
"ENTER"
patch.exe khpcpatchmanager kh1_third.kh1pcpatch
"ENTER"'

And I was of the assumption `Console.Readline();` was the culprit since it waits for a user input to proceed (might be wrong, like I said, I'm very new to C# as a whole)

**Comment by AntonioDePau** [comment]
Yes, but you can already apply multiple patches at once currently:
`KHPCPatchManager.exe myFirstPatch.kh1pcpatch mySecondPatch.kh1pcpatch myThirdPatch.kh1pcpatch`
And so on.

Let me know if that works for you or not.

**Comment by PerryTute** [comment]
ah crap I had no idea

> Yes, but you can already apply multiple patches at once currently: `KHPCPatchManager.exe myFirstPatch.kh1pcpatch mySecondPatch.kh1pcpatch myThirdPatch.kh1pcpatch` And so on.
> 
> Let me know if that works for you or not.

Oh sweet that works nicely, but would you at all know what change would need to be made to get it to terminate once its done with the patching process? I'm making an installer for some mod projects I'm working on, and I'm getting it to launch a batch script once its finished extracting the mods, to patch the mods in, and there as some optional components that can be included that require multiple lines like the one above, that hopefully can be automated and terminate once they're done patching, I hope this explains my reasoning for seeking such a solution.

**Comment by AntonioDePau** [comment]
Hey, sorry I took so long to reply.
If you can compile the tool yourself, you could simply comment out line 300 so that there is not key input required to terminate the patching process once done:
https://github.com/AntonioDePau/KHPCPatchManager/blob/main/KHPCPatchManager.cs#L300
`// if(!GUI_Displayed) Console.ReadLine(); // commented in order to skip user input once done`
